SR2021/2020-11-13-sr2021-kickstart-details.md

```diff
@@ -0,0 +1,28 @@
+---
+to: Potential team leaders + signed up teams
+subject: Student Robotics Kickstart Information
+---
+
+Hello there!
+
+We're pleased to announce some more details for the Student Robotics 2020 kickstart.
+
+As you know, following on from the success of our [previous virtual competition](https://studentrobotics.org/news/2020-07-25-post-competition/), we will be running the competition in a simulated environment this year.
+
+## Kickstart
+
+Kickstart will be streamed live on **21st November 2020 at 13:00 (UK Time)**, aiming to finish around 16:00. Here we'll announce the game for this year, go into further details on how the competition year will work, and of course answer any questions you have.
+
+If you've signed up for the competition, you'll receive the access details for your team. With these you can create user accounts for your competitors to give them access to the IDE and forum. We recommend creating accounts for competitors before kickstart to ensure they can get started straight away.
```

Comment:
Are we doing Discord as well as or instead of the forums? I had thought instead of.
Either way: we should update this to mention Discord - both that we're using it and how we'll send out the signup details.